libs/ledger-live-common/src/countervalues/api/api.ts

```diff
@@ -8,12 +8,15 @@ const baseURL = () => getEnv("LEDGER_COUNTERVALUES_API");
 
 const latest = async (pairs: TrackingPair[], direct?: boolean) => {
   const { data } = await network({
-    method: "POST",
-    url: `${baseURL()}/latest${direct ? "?method=direct" : ""}`,
-    data: pairs.map((p) => ({
-      from: p.from.countervalueTicker ?? p.from.ticker,
-      to: p.to.countervalueTicker ?? p.to.ticker,
-    })),
+    method: "GET",
+    url: `${baseURL()}/latest${direct ? "/direct" : "/indirect"}?pairs=${pairs
+      .map(
```

Comment:
there don't seem to be any chunking logic here, but maybe it's somewhere else?
Can you confirm it won't break if there are like 1000 pairs to fetch?